investigations/cava2/_CoqProject

```diff
@@ -0,0 +1,15 @@
+INSTALLDEFAULTROOT = ./
+-R ../../third_party/coq-ext-lib/theories ExtLib
+-R ../../third_party/bedrock2/deps/coqutil/src/coqutil coqutil
+-R . Cava
+-R ../../cava/Cava Cava
```

Comment:
Two paths for `Cava`?

Reply:
This is temporary so I can call the local module Cava but also access the old Cava at the other path, mostly to access Cava Lemmas.